doc/developer-notes.md

```diff
@@ -874,6 +875,22 @@ test/lint/commit-script-check.sh origin/master..HEAD
 
 Commit [`bb81e173`](https://github.com/bitcoin/bitcoin/commit/bb81e173) is an example of a scripted-diff.
 
+Release notes
+-------------
+
+Release notes should be written for any PR that:
+
+- introduces a notable new feature
+- fixes a significant bug
+- changes an API or configuration model
+- makes any other visible change to the end-user experience.
+
+Release notes should be added to the release notes file in
+[/doc/release-notes.md](/doc/release-notes.md) or, to avoid conflicts between
```

Comment:
Why do we suggest something that's going to break outside of users' control?

Reply:
Sorry, I don't understand the question.

Reply:
If you suggest this, people will do it, and things will break. Why not have them do the latter and make a new file?

Reply:
I think it's up to the individual contributor, but I'm happy to change this to just suggest using `release-notes-XXXXX.md`.

Reply:
I think a single actionable item is better than 2, one of which has negative externalities(maybe just me)

Reply:
IMO I'd leave both - the PR author has the right to update `doc/release-notes.md`, especially if he wants to work out existing text.

There's also the case where one PR XXX updates `release-notes-YYY.md`

> to avoid conflicts between multiple PRs.

IMO the conflicts in release notes shouldn't be postponed to release time.

Reply:
@promag they're typically not real conflicts, but "oops pushed a line down" conflicts.

Reply:
I think that's fine, that also happens with code and must be fixed accordingly. I'm not against individual files, both options are valid for me.

Anyway, looks like now I'm forced to write individual release notes. See #15193 for instance, would you NACK it because it doesn't create an individual file?

Reply:
Just my opinion, everyone else seems to disagree :)

Reply:
I don't want to make this prescriptive that you can't write notes in release-notes.md, and I certainly wouldn't want to encourage "please move release notes to a separate file" review comments. However, I agree with instagibbs that only presenting one action is better.

I'm happy to go either way.